trackers/class-wp-sentry-js-tracker.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+require_once __DIR__ . '/class-wp-sentry-tracker-base.php';
+
+/**
+ * WordPress Sentry Javascript Tracker.
+ */
+final class WP_Sentry_Js_Tracker extends WP_Sentry_Tracker_Base {
+
+	/**
+	 * Get the sentry tracker instance.
+	 *
+	 * @return WP_Sentry_Js_Tracker
+	 */
+	public static function get_instance() {
+		static $instance = null;
+		return $instance ?: $instance = new self( WP_SENTRY_PUBLIC_DSN );
+	}
+
+	/**
+	 * Register WordPress hooks.
+	 */
+	protected function register_hooks() {
+
+		// Call parent method.
+		parent::register_hooks();
+
+		// Register on front-end using the highest priority.
+		add_action( 'wp_enqueue_scripts', [ $this, 'on_enqueue_scripts' ], 0, 1 );
+
+		// Register on admin using the highest priority.
+		add_action( 'admin_enqueue_scripts', [ $this, 'on_enqueue_scripts' ], 0, 1 );
+
+		// Register on login using the highest priority.
+		add_action( 'login_enqueue_scripts', [ $this, 'on_enqueue_scripts' ], 0, 1 );
+	}
+
+	/**
+	 * Get sentry dsn.
+	 *
+	 * @return string
+	 */
+	public function get_dsn() {
+
```

Comment:
Bit of a perfectionist comment but I don't like the looks of this white line in each method.